package.json

```diff
@@ -59,6 +65,7 @@
     "through2": "^0.6.3"
   },
   "scripts": {
+    "start": "gulp server -o http://localhost:8080/dist/docs/index.html",
```

Comment:
It would also be nice if this command would auto-open the docs site in the browser. `gulp-connect` doesn't have that feature without using the `opn` package, but I'm not sure why we're using both `gulp-connect` and `gulp-webserver`. We should probably pick one.